### PR TITLE
Add support for preventAdditionalObjectProperties to jsbq cli

### PR DESCRIPTION
**Pull request by ahsandar**
Add support for igonreAdditional to jsbq cli.

### CODE REVIEW

**Comment by ahsandar** [comment]
@thedumbterminal please help review for merge. thanks

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Note that some features involve bespoke interpretation of schema details suited
 
 ## Consume
 
-    jsbq -p <gbq project> -d <gbq dataset> -j <json schema file>
+    jsbq -p <gbq project> -d <gbq dataset> -j <json schema file> --ignoreAdditional <boolean>
```
Comment (thedumbterminal) [inline README.md]: No need for `<boolean>` here.

File: bin/jsbq [hunk 2]
```diff
@@ -46,7 +46,7 @@ jsbq.run = async () => {
   const readFile = promisify(fs.readFile)
 
   const argv = require('yargs')
-  .usage('Usage: $0 -p [project] -d [dataset] -j <json schema file>')
+  .usage('Usage: $0 -p [project] -d [dataset] -j <json schema file> --ignoreAdditional <boolean>')
```
Comment (thedumbterminal) [inline bin/jsbq]: No need for `<boolean>` here.

File: bin/jsbq [hunk 3]
```diff
@@ -57,14 +57,19 @@ jsbq.run = async () => {
     },
     d: {
       describe: 'BigQuery dataset name'
+    },
+    ignoreAdditional: {
+      describe: 'skip checking for additional properties in schemas'
```
Comment (thedumbterminal) [inline bin/jsbq]: Can we add `type: 'boolean'` too.

Comment (thedumbterminal) [reply bin/jsbq]: And also `default: false`.

File: bin/jsbq [hunk 4]
```diff
@@ -57,14 +57,19 @@ jsbq.run = async () => {
...
     }
   })
   .argv
 
   const schemaData = await readFile(argv.j)
+  options = {};
```
Comment (thedumbterminal) [inline bin/jsbq]: This and the next line can simply now be:
```
const options = {
  ignoreAdditional: argv.ignoreAdditional
}
```

File: README.md (hunk 1 above)
Comment (ahsandar) [reply README.md]: updated

File: bin/jsbq (hunk 3 above)
Comment (ahsandar) [reply bin/jsbq]: updated

File: bin/jsbq (hunk 2 above)
Comment (ahsandar) [reply bin/jsbq]: updated

**Comment by ahsandar** [comment]
> Can we change the option to match the new name, thanks for this work!

updated

File: bin/jsbq (hunk 4 above)
Comment (ahsandar) [reply bin/jsbq]: updated

**Comment by ahsandar** [comment]
@thedumbterminal updated based on review feedback, please chk, thanks

**Comment by ahsandar** [comment]
thanks

**Pull request merged**